src/Avalonia.Themes.Fluent/FluentTheme.cs

```diff
@@ -58,6 +64,18 @@ public FluentThemeMode Mode
             get => GetValue(ModeProperty);
             set => SetValue(ModeProperty, value);
         }
+        
+        public static readonly StyledProperty<DensityStyle> DensityStyleProperty =
+            AvaloniaProperty.Register<FluentTheme, DensityStyle>(nameof(DensityStyle));
```

Comment:
i suggest moving it higher to be together with other styled properties

Reply:
Addressed this and other comments